content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -14,11 +14,13 @@ test: no
 ---
 
 Setup a [multicluster Istio service mesh](/docs/ops/deployment/deployment-models/#multiple-clusters)
-across multiple clusters with a shared control plane. In this configuration, multiple Kubernetes clusters running
-a remote configuration connect to a shared Istio [control plane](/docs/ops/deployment/deployment-models/#control-plane-models)
-running in a main cluster. Clusters may be on the same network or different networks than other
-clusters in the mesh. Once one or more remote Kubernetes clusters are connected to the Istio control plane,
-Envoy can then form a mesh.
+across multiple {{< gloss "remote cluster" >}}remote clusters{{< /gloss >}} that
+share a control plane. In this configuration, multiple Kubernetes clusters
+running a remote configuration connect to a shared Istio [control plane](/docs/ops/deployment/deployment-models/#control-plane-models)
+running in a {{< gloss >}}primary cluster{{< /gloss >}}. Remote clusters can be
+in the same network or on a different network as the primary clusters. After one
+or more remote clusters are connected, the control plane manages the service
```

Comment:
I think we can drop the last sentence, it is confusing.

Reply:
Added links to the glossary and specificity around where the control plane resides for additional clarity.
Further rewrites are outside the scope of this PR.